tests/host_tools/network.py

```diff
@@ -59,6 +59,23 @@ def scp_file(self, local_path, remote_path):
         else:
             utils.run_cmd(cmd)
 
+    def scp_get_file(self, remote_path, local_path):
+        """Copy a files to the VM using scp."""
```

Comment:
should it be `"""Copy a file from the VM using scp"""`? I see that there is already a function in this file that handles copying to the VM